src/components/NavigationDesktop/NavigationItemDesktop.js

```diff
@@ -116,20 +127,36 @@ class NavigationItemDesktop extends Component {
           open={this.state.isSubNavOpen}
         >
           <Grid container className={classes.grid} spacing={16}>
-            {items.map((item, index) => (
-              <Grid item key={index}>
-                <MenuList disablePadding>
-                  <MenuItem>
-                    <Link onClick={this.onClose} href={this.linkPath(item)} linkItem={item}>
-                      <ListItemText primary={item.navigationItem.data.contentForLanguage} />
-                    </Link>
-                  </MenuItem>
-                  {Array.isArray(item.items) && this.renderSubNav(item)}
-                </MenuList>
-              </Grid>
-            ))}
+            {items.map((item, index) => {
+              const { navigationItem: { data: { contentForLanguage, classNames: navigationItemClassNames, isUrlRelative, shouldOpenInNewWindow } } } = item;
+              return (
+                <Grid item key={index}>
+                  <MenuList disablePadding>
+                    <MenuItem>
+                      <Link
+                        className={navigationItemClassNames}
+                        href={this.linkPath(item)}
+                        isUrlRelative={isUrlRelative}
+                        onClick={this.onClose}
+                        shouldOpenInNewWindow={shouldOpenInNewWindow}
+                      >
+                        <ListItemText primary={contentForLanguage} />
+                      </Link>
+                    </MenuItem>
+                    {Array.isArray(item.items) && this.renderSubNav(item)}
+                  </MenuList>
+                </Grid>
+              );
+            })}
           </Grid>
-          <Link className={classes.navigationShopAllLink} onClick={this.onClose} route={`${this.linkPath()}`} linkItem={navItem}>
+          <Link
+            className={classes.navigationShopAllLink}
+            onClick={this.onClose}
+            route={`${this.linkPath()}`}
```

Comment:
This doesn't need to be in a string template, and should `navItem` be passed in like the `href`?